docs/user-guide/ui.md

```diff
@@ -2,76 +2,68 @@
 
-Dashboard also provides information on the state of Pods, Replication controllers, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources, as well as your deployed containerized applications. You can also change the number of replicated Pods, delete Pods, and deploy new applications using a deploy wizard.
+Dashboard also provides information on the state of Deployments, Replica Sets, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources and applications deployed on the cluster. For example, you can scale a Deployment, initiate a rolling update, restart a pod or deploy new applications using a deploy wizard.
 
-By default, Dashboard is installed as a cluster addon. It is enabled by default as of Kubernetes 1.2 clusters.
+![Kubernetes Dashboard UI](/images/docs/ui-dashboard.png)
 
 * TOC
 {:toc}
 
-## Dashboard access
+## Accessing Dashboard UI
 
-Navigate in your Browser to the following URL:
-```
-https://<kubernetes-master>/ui
-```
-This redirects to the following URL:
-```
-https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
-```
-The Dashboard UI lives in the `kube-system` [namespace](/docs/admin/namespaces/), but shows all resources from all namespaces in your environment.
-
-If you find that you are not able to access Dashboard, you can install and open the latest stable release by running the following command:
+### Command line proxy
+The easiest way to access Dashboard is to use kubectl. Run the following command in your desktop environment:
 
 ```
-kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
+$ kubectl proxy
 ```
 
-Then, navigate to
+kubectl will handle authentication with apiserver and make Dashboard available at [http://localhost:8001/ui](http://localhost:8001/ui)
 
-```
-https://<kubernetes-master>/ui
-```
+The UI can _only_ be accessed from the machine where the command is executed. See `kubectl proxy --help` for more options.
 
-In case you have to provide a password, use the following command to find it out:
+### Master server
+You may access the UI directly via the Kubernetes master apiserver. Open a browser and navigate to `https://<kubernetes-master>/ui`, where `<kubernetes-master>` is IP address or domain name of the Kubernetes
+master.
 
-```
-kubectl config view
-```
+Please note, this works only if the apiserver is set up to allow authentication with username and password. This is not currently the case with the some setup tools (e.g., `kubeadm`). Refer to the  [documentation](docs/admin/authentication/) to configure it manually.
```

Comment:
"to configure it manually" -> "for information on how to configure authentication manually."